Actions/CreateApp/CreateApp.ps1

```diff
@@ -3,6 +3,10 @@ Param(
     [string] $actor,
     [Parameter(HelpMessage = "The GitHub token running the action", Mandatory = $false)]
     [string] $token,
+    [Parameter(HelpMessage = "Specifies the parent telemetry scope for the Telemetry signal", Mandatory = $false)]
```

Comment:
Tests are failing because telemetryEventId is not as a parameter in the yaml file